api/alembic/versions/839f18e0ecc4_insert_c7b.py

```diff
@@ -0,0 +1,48 @@
+"""Insert C7B
+
+Revision ID: 839f18e0ecc4
+Revises: 871c39cf6c26
+Create Date: 2022-02-01 13:54:20.342071
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '839f18e0ecc4'
+down_revision = '871c39cf6c26'
+branch_labels = None
+depends_on = None
+
+
+c7b = 'C7B'
+stations_with_c7b = (305, 346, 286, 298, 328, 334, 1399, 836, 280, 1055, 1029, 309, 306)
```

Comment:
this could be problematic because I know for sure that 1055 (and possibly other stations) are used in different planning areas in diff fire centres, and I think that sometimes the fuel type selected for the station may be different for diff planning areas

Reply:
the Fraser zone in Coastal also uses 1055 - have you confirmed that they also want C7B?

Reply:
Thanks @andrea-williams ! What a grand error.